hotel-service/src/main/java/edu/myhotel/hotel/service/storage/DataStorageStub.java

```diff
@@ -0,0 +1,130 @@
+                                    RoomFacility.TEAPOT, RoomFacility.COFFEE_MACHINE,
+                                    RoomFacility.IRON, RoomFacility.HAIRDRYER,
+                                    RoomFacility.MINIBAR, RoomFacility.TV));
+
+        Room room5 = new Room();
+        room5.setId(5);
+        room5.setType(RoomType.DOUBLE);
+        room5.setName("Friday");
+        room5.setPrice(1400);
+        room5.setImage("https://via.placeholder.com/100");
+        room5.setFacilities(List.of(RoomFacility.DOUBLE_BED, RoomFacility.SAFE,
+                                    RoomFacility.BATHTUB, RoomFacility.TOWELS,
+                                    RoomFacility.WIFI, RoomFacility.AIR_CONDITION,
+                                    RoomFacility.TEAPOT, RoomFacility.COFFEE_MACHINE,
+                                    RoomFacility.IRON, RoomFacility.HAIRDRYER,
+                                    RoomFacility.MINIBAR, RoomFacility.TV));
+
+        Room room6 = new Room();
+        room6.setId(6);
+        room6.setType(RoomType.QUAD);
+        room6.setName("Saturday");
+        room6.setPrice(2400);
+        room6.setImage("https://via.placeholder.com/100");
+        room6.setFacilities(List.of(RoomFacility.DOUBLE_BED, RoomFacility.TWIN_BED,
+                                    RoomFacility.TWIN_BED,RoomFacility.SAFE,
+                                    RoomFacility.BATHTUB, RoomFacility.SHOWER, RoomFacility.TOWELS,
+                                    RoomFacility.WIFI, RoomFacility.AIR_CONDITION,
+                                    RoomFacility.TEAPOT, RoomFacility.COFFEE_MACHINE,
+                                    RoomFacility.IRON, RoomFacility.HAIRDRYER,
+                                    RoomFacility.MINIBAR, RoomFacility.TV));
+
+        Room room7 = new Room();
+        room7.setId(7);
+        room7.setType(RoomType.SUITE);
+        room7.setName("Sunday");
+        room7.setPrice(3000);
+        room7.setImage("https://via.placeholder.com/100");
+        room7.setFacilities(List.of(RoomFacility.DOUBLE_BED, RoomFacility.DOUBLE_BED,
+                                    RoomFacility.TWIN_BED, RoomFacility.TWIN_BED, RoomFacility.SAFE,
+                                    RoomFacility.BATHTUB, RoomFacility.SHOWER, RoomFacility.TOWELS,
+                                    RoomFacility.WIFI, RoomFacility.AIR_CONDITION,
+                                    RoomFacility.TEAPOT, RoomFacility.COFFEE_MACHINE,
+                                    RoomFacility.IRON, RoomFacility.HAIRDRYER,
+                                    RoomFacility.MINIBAR, RoomFacility.TV));
+
+        rooms.add(room1);
+        rooms.add(room2);
+        rooms.add(room3);
+        rooms.add(room4);
+        rooms.add(room5);
+        rooms.add(room6);
+        rooms.add(room7);
+    }
+
+    @Override
+    public List<Room> findAvailableRooms(LocalDate checkIn, LocalDate checkOut) {
+        return rooms;
```

Comment:
Please add stubs for `bookedRooms` and implement searching available rooms for a given dates.